ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/ledger/ScenarioLoader.scala

```diff
@@ -116,12 +119,19 @@ private[sandbox] object ScenarioLoader {
     val scenarioQualName = getScenarioQualifiedName(packages, scenario)
     val candidateScenarios = getCandidateScenarios(packages, scenarioQualName)
     val (scenarioRef, scenarioDef) = identifyScenario(packages, scenario, candidateScenarios)
-    @com.github.ghik.silencer.silent("can be used only by sandbox classic.")
-    val scenarioLedger =
-      ScenarioRunner.getScenarioLedger(engine, scenarioRef, scenarioDef, transactionSeed)
+    val scenarioLedger = getScenarioLedger(engine, transactionSeed, scenarioRef, scenarioDef)
     (scenarioLedger, scenarioRef)
   }
 
+  //noinspection ScalaDeprecation
```

Comment:
What does this do?

Reply:
IntelliJ suppression I added by accident. I'll remove it. (IntelliJ should be able to understand `@nowarn`, but Scala support is a little lacking.)